src/main/java/net/barrage/school/java/ecatalog/app/productSource/ProductSourceService.java

```diff
@@ -0,0 +1,13 @@
+package net.barrage.school.java.ecatalog.app.productSource;
+
+public interface ProductSourceService {
```

Comment:
minor: This name looks confusing for me. Cause here we have 2 entities: source and db, and we sync them. IMHO `ProductSyncService` or `ProductDbSyncService` will be much clear.